vertica-kafka-scheduler/README.md

```diff
@@ -248,8 +248,8 @@ vkconfig microbatch --add \
 After you [create a scheduler](#create-a-scheduler), launch the scheduler to begin scheduling microbatches. To launch a scheduler, execute a `docker run` command that does the following:
 - Mounts a configuration file as a volume.
 - Specifies the scheduler image name.
-- Mounts `vkafka-log-config-debug.xml` from the current directory file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
-- Mounts `/log` from the current directory into the container's `/opt/vertica/log` directory so that log messages are written to the local `/log` directory.
+- Mounts the `vkafka-log-config-debug.xml` file in the current directory into the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
+- Mounts `log` in the current directory into the container as the `/opt/vertica/log` directory. This writes log messages to the local `log` directory.
```

Comment:
Now I'm being picky, but I'd like to see these two mount sentences look the same grammatically.  The first sentence is about mounting a file and the second is about mounting a directory, so they aren't identical, but they should be closer wrt "into", "the container's", "as the", etc.

And as long as you are making changes here, the "This writes log messages" should maybe be "This allows log messages to be written to" ... "assuming `vkafka-log-config-debug.xml` still configures log output in `/opt/vertica/log`

Reply:
I thought the same thing about the sentence structure, but thought maybe it was more clear bc they were mounting different things. I updated the structure to be the same, let me know what you think.

Reply:
I like it.  Thanks!